Makefile

```diff
@@ -55,9 +55,9 @@ nupkg: lib lib-win
 	mv $(DOTNET_DIR)/build/SecretHub.*.nupkg .
 	make clean
 
-#.PHONY: nupkg-publish
-#nupkg-publish: nupkg
-	#dotnet nuget push *.nupkg --api-key <API_KEY> --source 	https://api.nuget.org/v3/index.json
+.PHONY: nupkg-publish
+nupkg-publish: nupkg
+	dotnet nuget push SecretHub.*.nupkg --api-key ${API_KEY} --source https://api.nuget.org/v3/index.json
```

Comment:
Can I publish a new release as well? Because I don't see this variable declared anywhere in the makefile. It can be a secretpath and only those who have access to it can publish.

Reply:
Yep, we should fix this, but it does not block a merge right now.

Reply:
Oke then. To be considered in the next development phase.